### PR TITLE
Recipient & Transfer docs

### PR DESCRIPTION
**Pull request by tommy-omise**
- Add Recipient API docs
- Add Transfer documentation (quick guide)

### CODE REVIEW

**Comment by ppp-master** [comment]
I adjusted a few things and added the API call to generate a sample recipient for the docs.

Also: api/transfers/create_with_recipient is missing .java and .js files. I created empty placeholders.

File: source/transferring-funds.md.erb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+title: Transferring funds
+description: This page will guide you how to transfer the available balance to your account or third party recipient
+---
+
+## Requirement
+
+The following requirements must be met for a transfer to be created:
+
+- There must be over 30 THB in your available balance.
+- There cannot be another incomplete transfer being processed.
```
Comment (ppp-master) [inline source/transferring-funds.md.erb]: Is that still the case?

Comment (robinclart) [inline source/transferring-funds.md.erb]: Nope

Comment (tommy-omise) [inline source/transferring-funds.md.erb]: not anymore, transfer can be created anytime. But the transfer amount must not less than 30 THB.

**Pull request merged**